Documentation/kubernetes-on-aws.md

```diff
@@ -3,10 +3,20 @@
 Deploy a fully-functional Kubernetes cluster using AWS CloudFormation.
 Your cluster will be configured to use AWS features to enhance Kubernetes.
 For example, Kubernetes may automatically provision an Elastic Load Balancer for each Kubernetes Service.
-After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
-
 At CoreOS, we use the [kube-aws](https://github.com/coreos/coreos-kubernetes/releases) CLI tool to automate cluster deployment to AWS.
 
+After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the `kubectl` CLI tool.
+
+Each of the steps will cover:
+
+| Step | Description |
```

Comment:
I don't think this has to be a table. I think you could get your layout with list-sublist and it will be easier to maintain.

* [Step 1: Configure]() (this document)
  * Download the kube-aws CloudFormation generator
  * Define account and cluster settings
* [Step 2: Render]()
  * Compile a re-usable CloudFormation template for the cluster
  * Optionally adjust template configuration
  * Validate the rendered CloudFormation stack
* [Step 3: Spin up]()
  * ...